src/main/java/com/epam/esm/persistence/dao/GiftCertificate.java

```diff
@@ -0,0 +1,28 @@
+package com.epam.esm.persistence.dao;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class GiftCertificate {
+    private Long id;
+    private String name;
+    private String description;
+    @Builder.Default
+    private Double price = 0.0;
+    @Builder.Default
+    private Integer duration = 0;
```

Comment:
It's better to store duration data using class `Duration`. Storing in `Integer` is very confusing.